config/samples/federation/posts-subgraph.yaml

```diff
@@ -31,13 +31,13 @@ spec:
         version: v1
     spec:
       containers:
-        - image: zalbiraw/go-api-test-service
+        - image: zalbiraw/go-api-test-service:v2.0.0
```

Comment:
Can you please update services to `v1.0.0`. Because we are using hardcoded SDL and Schema right now and these fields are compatible with v1.0.0.